dashboard/app/main.go

```diff
@@ -1535,6 +1535,7 @@ func handleTextImpl(c context.Context, w http.ResponseWriter, r *http.Request, t
 	// https://bugs.chromium.org/p/chromium/issues/detail?id=608342
 	w.Header().Set("Content-Disposition", "inline; filename="+textFilename(tag))
 	augmentRepro(c, w, tag, bug, crash)
+	data = report.SplitReportBytes(data)[0]
```

Comment:
I'd suggest that we upload just the first report to the dashboard until we have figured out how we want to represent/store it on the web dashboard.

In its current form it won't prevent sending out emails with all reports combined.

Reply:
Because there is a code that fetches these records directly from DB?

Reply:
Found 2 points where we send reports and did the split there. PTAL.